pinot-core/src/main/java/com/linkedin/pinot/core/segment/DefaultSegmentNameGenerator.java

```diff
@@ -77,7 +77,7 @@ public DefaultSegmentNameGenerator(String timeColumnName, String tableName, Stri
    * @throws Exception
    */
   @Override
-  public String generateSegmentName(ColumnStatistics statsCollector) throws Exception {
+  public String generateSegmentName(ColumnStatistics statsCollector, String simpleDateFormat) throws Exception {
```

Comment:
Would be better to take the timeColumnFormat in the constructor along with timeColumnName.